### ISSUE TITLE
Images not being deployed in action

### ISSUE THREAD

**Issue by piksel** [containrrr/containrrr.dev#2]
For some reason, the images are not being included in the commit/deploy of the generate hugo site. I have not looked in to why this is happening and instead just push the images manually after deploy.

**Comment by simonaronsson** [comment]
Static assets like images are copied from `/static`,  not `/resources`. 👍

**Comment by piksel** [comment]
Yeah, you can add unmanaged resources to `/static` (and that's a better temporary solution). But these should be included as [page resources](https://gohugo.io/content-management/page-resources/) and that works correctly using `serve`, but not when building (at least not in CI)

Edit: Although, the main logotype is definitely in the wrong folder. It should just be moved to static.
